src/store/state/state-model.ts

```diff
@@ -217,5 +217,7 @@ export const stateModel: StateModelIF = {
     shareStructure: false,
     incorporationAgreement: false
   },
-  summaryMode: false
+  summaryMode: false,
+  currentFees: emptyFees,
```

Comment:
I gave you an example with `clone()` and told you why you should use it. Why didn't you use it?

Reply:
Yes, done now. Sorry I was still revising the comments.